docs/how-to/building.md

```diff
@@ -0,0 +1,231 @@
+`BINUTILS_DIR`:
+    This is the path to the gcc cross-compiler toolchain to use. By default this is set to `$(SDK_ROOT)/gnu/arm/lin`. The directory `$(BINUTILS_DIR)/bin` will be added to the path during the build.
+`SYSROOT`:
+    This is the path to the compiled system root (derived from libc). Normally this is part of the installed toolchain, and if not set elsewhere this is set to `$(BINUTILS_DIR)/$(COMPILER_PREFIX)/libc`.
+
+Note that the `COMPILER_PREFIX` symbol can also be overwritten if required. The default value is `arm-xilinx-linux-gnueabi`.
+
+### Sources
+
+The rootfs requires a long list of dependencies that are freely available on the web. They have been gathered together in one tar file here:
+
+[](http://www.ohwr.org/attachments/download/5133/rootfs-tarfiles-0.6.tar)
+
+They consist of the following sources, listed below with their MD5 checksums:
+
+```
+
+
+930d126df2113221e63c4ec4ce356f2c linux-xilinx-v2015.1.tar.gz
+b6d212208b7694f748727883eebaa74e u-boot-xlnx-xilinx-v2015.1.tar.gz
+
+82d05e03b93e45f5a39b828dc9c6c29b autoconf-2.69.tar.gz
+716946a105ca228ab545fc37a70df3a3 automake-1.15.tar.gz
+0b65a216ce9dc9c1a7e20a729dd7c05b backports_abc-0.4.tar.gz
+788214f20214c64631f0859dc79f23c6 backports.ssl_match_hostname-3.4.0.2.tar.gz
+7925683d7dd105aabe9b6b618d48cc73 busybox-1.23.2.tar.bz2
+bb13834970c468f73415618437f3feac conserver-8.2.0.tar.gz
+5d69a1b712fb8fec6ad461e676bf1097 cothread-2.14.tar.gz
+e967e320344cd4bfebe321e3ab8514d6 dropbear-2015.67.tar.bz2
+bc759fc62666786f5436e2075beb3265 e2fsprogs-1.42.13.tar.gz
+ac80f432ac9373e7d162834b264034b6 enum34-1.0.4.tar.gz
+3b2322695e9ee7bf447ebcdb85f93e83 ethtool-2.6.36.tar.gz
+5154c00201d599acc00194c6c278ca23 iperf-3.0.2.tar.gz
+277e4bd258fd4fb2aadaed760320c566 libressl-2.2.0.tar.gz
+addf44b646ddb4e3919805aa88fa7c5e libtool-2.4.6.tar.gz
+1b29c10db4aa88afcaeeaabeef6790db lsof_4.88.tar.bz2
+a5e9954b1dae036762f7b13673a2cf76 m4-1.4.17.tar.gz
+4ad1f758d49615efe14edb107eddac5c mtd-utils-1.5.1.tar.bz2
+1c612b478f976abf8ef926480c7a3684 nano-2.4.1.tar.gz
+8cb9c412e5f2d96bc6f459aa8c6282a1 ncurses-5.9.tar.gz
+fa37049383316322d060ec9061ac23a9 ntp-4.2.8p2.tar.gz
+a1ed53432dbcd256398898d35bc8e645 numpy-1.9.2.tar.gz
+aa3c86e67551adc3ac865160e34a2a0d pkg-config-0.28.tar.gz
+bbf052e7fcc6fa403d2514219346da04 procServ-2.6.0.tar.gz
+d7547558fd673bd9d38e2108c6b42521 Python-2.7.10.tgz
+33c8fb279e981274f485fd91da77e94a readline-6.3.tar.gz
+419a0594e2b25039239af8b90eda7d92 screen-4.2.1.tar.gz
+af2fc6a3d6cc5a02d0bf54d909785fcb singledispatch-3.4.0.3.tar.gz
+107a5be455493861189e9b57a3a51912 strace-4.10.tar.xz
+d13a99dc0b60ba69f5f8ec1235e5b232 tornado-4.3.tar.gz
+44d667c142d7cda120332623eab69f40 zlib-1.2.8.tar.gz
+```
+
+These packages are used as follows in the build:
+
+autoconf, automake, libtool, m4, pkg-config:
+
+    The rootfs builder needs specific and up to date versions of these tools.
```

Comment:
The way this appears on the published page looks a bit odd. I think it should be formatted more similarly to the OUTPUT FILES section